app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  # This is a module, part of ActiveModel library
```

Comment:
Thanks for these comments guys, super useful!

Reply:
👍